sources/metricdog/src/metricdog.rs

```diff
@@ -156,3 +157,29 @@ impl Metricdog {
         Ok(())
     }
 }
+
+/// reqwest needs a URL protocol scheme to be present, but other implementations assume
+/// `http://` as the scheme when none is present. We need to check the `HTTPS_PROXY` env and
+/// reset it with `http://` when no scheme is present.
+// TODO - remove this if reqwest adds support for proxy addresses with no scheme.
```

Comment:
Same comment as I left on https://github.com/bottlerocket-os/bottlerocket/pull/1324, can you link to https://github.com/seanmonstar/reqwest/issues/1176 here?

Reply:
I thought it better to link to a tracking issue since it also has to go through tough and is wrapped up in the update to everything tokio v1